### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by gallmann-ubique**

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -100,6 +100,6 @@ dependencies {
 	api 'com.google.zxing:core:3.4.1'
 
 	testImplementation 'junit:junit:4.+'
```
Comment (M-Wong) [inline common/build.gradle:102]: Could we change this to a fixed version? Not that it should be a problem with JUnit, but in general I prefer not having unstable dependencies (and also getting rid of the Android Studio lint warning would be nice :smile: )
Do we even need it here? We don't have any unit tests in the common module and the wallet has its own testing dependencies

File: build.gradle [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
-	ext.kotlin_version = "1.5.30"
+	ext.kotlin_version = "1.6.10"
 	repositories {
 		google()
 		mavenCentral()
 	}
 	dependencies {
-		classpath 'com.android.tools:r8:3.0.69'
-		classpath 'com.android.tools.build:gradle:7.0.3'
+		classpath 'com.android.tools:r8:3.0.73'
```
Comment (M-Wong) [inline build.gradle:19]: I think we included this version due to a bug in R8 that made problems with the reproducible builds. Could you check with @goebelUB whether this version contains the fix?

Comment (goebelUB) [reply build.gradle:19]: 3.0.69 included the fix, so I would hope that it's still there in 3.0.73 :)

Sidenote: we only included the r8 dependency to manually override whatever r8 version the Android toolchain ships with. If at some point the toolchain comes with a fixed version as well, we could remove the explicit r8 dependency alltogether.

Comment (M-Wong) [reply build.gradle:19]: Perfect, thanks :smiley:

**Review by M-Wong — APPROVED** [review]

**Review by benz-ubique — APPROVED** [review]

**Pull request merged**